yt/utilities/object_registries.py

```diff
@@ -0,0 +1,7 @@
+# These are some of the data object registries that are used in different places in the code.  Not all of the self-registering objects are included in these.
```

Comment:
this line violates flake8 E501 (line too long) that I'm trying to reintroduce in #2796
```suggestion
# These are some of the data object registries that are used in different places in the code. 
# Not all of the self-registering objects are included in these.
```

Reply:
Oops, this was my mistake; I actually word-wrapped it and then messed up with my changeset.